lib/iex/lib/iex/helpers.ex

```diff
@@ -328,6 +328,21 @@ defmodule IEx.Helpers do
 
   Use negative values to look up expression values relative to the current one.
   For instance, v(-1) returns the result of the last evaluated expression.
+
+  ## Examples
+
+      iex(1)> "hello" <> " world"
+      "hello world"
+      iex(2)> 40 + 2
+      42
+      iex(3)> foo = "bar"
+      "bar"
+      iex(4)> v(-2)
+      42
+      iex(5)> v(3)
+      "bar"
+      iex(6)> str = v()
```

Comment:
I would drop the binding here, and just show what `v()` does (especially since we're not using `str` later on either).